cli/src/services/session.service.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { SessionService } from './session.service';
+import mockfs from 'mock-fs';
+import fs from 'node:fs';
+import yaml from 'yaml';
+import { LoginError } from '../cores/errors/login-error';
+
+const mockPingServer = jest.fn(() => Promise.resolve({ data: { res: 'pong' } }));
+const mockUserInfo = jest.fn(() => Promise.resolve({ data: { email: 'admin@example.com' } }));
+
+jest.mock('../api/open-api', () => {
+  return {
+    __esModule: true,
+    ...jest.requireActual('../api/open-api'),
+    UserApi: jest.fn().mockImplementation(() => {
```

Comment:
You can also use `mockReturnValue({ getMyUserInfo: mockUserInfo })` I believe.

Reply:
No, that doesn't work since the init order reverses